Gemfile

```diff
@@ -71,4 +74,7 @@ group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem "capybara"
   gem "selenium-webdriver"
+  gem 'factory_bot_rails'
+  gem 'faker'
+    gem 'shoulda-matchers', '~> 5.0'
```

Comment:
@NamraAfzal `faker` gem should be in [:development, :test] group. As we need this gem only for testing and development. `factory_bot_rails` and `shoulda-matchers` should be in test group.